crates/ruff_linter/src/codes.rs

```diff
@@ -403,6 +403,7 @@ pub fn code_to_rule(linter: Linter, code: &str) -> Option<(RuleGroup, Rule)> {
         (Flake8Quotes, "001") => (RuleGroup::Stable, rules::flake8_quotes::rules::BadQuotesMultilineString),
         (Flake8Quotes, "002") => (RuleGroup::Stable, rules::flake8_quotes::rules::BadQuotesDocstring),
         (Flake8Quotes, "003") => (RuleGroup::Stable, rules::flake8_quotes::rules::AvoidableEscapedQuote),
+        (Flake8Quotes, "100") => (RuleGroup::Stable, rules::flake8_quotes::rules::UnnecessaryEscapedQuote),
```

Comment:
Should this rule go into `RuleGroup::Preview`? One on hand it's a new rule, but on the other hand its purpose is fixing a bug in another rule...

Reply:
Yeah, it does need to go in preview -- as part of our [versioning policy](https://docs.astral.sh/ruff/versioning/), we only add new stable rules in minor (not patch) releases.

Reply:
I know it's a bit awkward, but basically, if we implemented this as an improvement in the `Q001` fix, it would be fine to ship as part of stable. But if we're adding a new rule, which can raise new violations, we want to avoid promoting it outside of a minor release.